### PR TITLE
Fix compilation error on Windows

### PR DESCRIPTION
**Pull request by yogwoggf**
# Changes

- Explicitly casts the path parts used in `fileExists` to strings
- Explicitly casts the path parts used in `getFileMetadata` to strings

These changes allow compilation to proceed on many compilers for Windows.

### CODE REVIEW

**Comment by yogwoggf** [comment]
TASBox will also need to pull the latest VPKParser after this is merged to fix the compile error there

**Review by Derpius — APPROVED** [review]
Lgtm (assuming you test locally before merging)

**Comment by yogwoggf** [comment]
Tested

**Pull request merged**
